tests/sepp/ValueGenerators/RandomStringGenerator.cpp

```diff
@@ -0,0 +1,74 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2022 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Manuel Pöter
+////////////////////////////////////////////////////////////////////////////////
+
+#include "RandomStringGenerator.h"
+
+#include <cmath>
+#include <cstdint>
+#include <string>
+#include <string_view>
+
+#include "Basics/debugging.h"
+#include "velocypack/Builder.h"
+
+namespace {
+constexpr std::string_view charset =
+    "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.,";
+
+}
+
+namespace arangodb::sepp::generators {
+
+RandomStringGenerator::RandomStringGenerator(std::uint32_t size) : _size(size) {
+  _prng.seed(0, 0xdeadbeefdeadbeefULL);
```

Comment:
Seeding like a boss.

Reply:
Actually there should be a TODO to seed properly so that different threads use different (but deterministic seeds). 🤔

Reply:
Ok, will you add the TODO here, so we don't forget about it?